src/main/java/slimeknights/tconstruct/library/events/TinkerCraftingEvent.java

```diff
@@ -25,7 +25,7 @@
  */
 @Cancelable
 public class TinkerCraftingEvent extends TinkerEvent {
-  private final ItemStack itemStack;
+  private ItemStack itemStack;
```

Comment:
No, the events do not allow to change the outcome, this needs to stay immutable.